modules/33acrossBidAdapter.js

```diff
@@ -83,6 +83,14 @@ function _createServerRequest(bidRequest, gdprConsent) {
       gdpr: (gdprConsent.gdprApplies === true) ? 1 : 0
     }
   };
+  ttxRequest.ext = {
+    ttx: {
+      caller: {
```

Comment:
I understand that we will not be tracking adapter version as well, but can still send caller as an Array like so:
```
caller: { [
   'name': 'prebidjs',
   'version': '$prebid.version$'
] }
```

This is because TTX expects caller to be an array to document a chain of callers (as in S2S reqs).

Reply:
Sure, np!